pkgs/build-support/cc-wrapper/add-hardening.sh

```diff
@@ -81,8 +81,15 @@ for flag in "${!hardeningEnableMap[@]}"; do
       hardeningCFlags+=('-fPIC')
       ;;
     strictoverflow)
-       if (( "${NIX_DEBUG:-0}" >= 1 )); then echo HARDENING: enabling strictoverflow >&2; fi
-      hardeningCFlags+=('-fno-strict-overflow')
+      if (( "${NIX_DEBUG:-0}" >= 1 )); then echo HARDENING: enabling strictoverflow >&2; fi
+      if (( @isClang@ )); then
+        # In Clang, -fno-strict-overflow only serves to set -fwrapv and is
+        # reported as an unused CLI argument if -fwrapv or -fno-wrapv is set
+        # explicitly, so we side step that by doing the conversion here.
+        hardeningCFlags+=('-fwrapv')
```

Comment:
```suggestion
        # In Clang, -fno-strict-overflow only serves to set -fwrapv and is
        # reported as an unused CLI argument if -fwrapv or -fno-wrapv is set
        # explicitly, so we side step that by doing the conversion here.
        # https://github.com/llvm/llvm-project/blob/dae52dd07d633673fe3ff2a6b5e5c71f9affc470/clang/lib/Driver/ToolChains/Clang.cpp#L6450-L6451
        hardeningCFlags+=('-fwrapv')
```

Reply:
Or a better link if you can find one.

Reply:
Thank you for your review. I appreciate it.